test/test_genbank_sbol3_direct.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import sbol3
+from sbol_utilities.sbol_diff import doc_diff
+from sbol_utilities.sbol3_genbank_conversion import convert_genbank_to_sbol3, \
+    GENBANK_FILE_1, SBOL3_FILE_1
+
+class TestGenBank2SBOL3(unittest.TestCase):
+    def test_simple_file_1(self):
+        """Test conversion of a simple genbank file with a single sequence"""
+        sbol3.set_namespace("https://testing.sbol3.genbank/")
+        TEST_OUTPUT_SBOL3 = SBOL3_FILE_1 + ".test" 
+        # Don't write to file for testing, we directly compare sbol documents
+        test_output_sbol3 = convert_genbank_to_sbol3(GENBANK_FILE_1, TEST_OUTPUT_SBOL3, write=False)
+        sbol3_file_1 = sbol3.Document()
+        sbol3_file_1.read(location=SBOL3_FILE_1, file_format=sbol3.SORTED_NTRIPLES)
+        assert not doc_diff(test_output_sbol3, sbol3_file_1), \
```

Comment:
unnecessary backslash here as well

Reply:
For this case, actually it's outside parentheses, and the linter gives an error on removing it.